src/js/view/controls/controlbar.js

```diff
@@ -277,7 +272,13 @@ export default class Controlbar {
         _model.change('state', this.onState, this);
         _model.change('duration', this.onDuration, this);
         _model.change('position', this.onElapsed, this);
-        _model.change('fullscreen', this.onFullscreen, this);
+        _model.change('fullscreen', (model, val) => {
+            toggleClass(this.elements.fullscreen.element(), 'jw-off', val);
```

Comment:
`const fullscreenElement = this.elements.fullscreen.element();` for minification